app/controllers/AsyncController.scala

```diff
@@ -22,7 +22,7 @@ object AsyncController extends Controller {
   private def simpleZ3Check(): Future[String] = Future {
     val config = new util.HashMap[String, String]()
     config.put("model", "true")
-    config.put("unsat_core", "true")
+    config.put("unsat_core", "true")  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/app/controllers/AsyncController.scala
+++ b/app/controllers/AsyncController.scala
@@ -22,7 +22,7 @@
   private def simpleZ3Check(): Future[String] = Future {
     val config = new util.HashMap[String, String]()
     config.put("model", "true")
-    config.put("unsat_core", "true")  
+    config.put("unsat_core", "true")
     val context = new Context(config)
     val a = 4
     val b = 8
```